Makefile

```diff
@@ -54,7 +56,8 @@ package:
 	$(MAKE) -C dist package VERDICTD_VERSION="$(VERDICTD_VERSION)" VERDICTD_MAINTAINER="$(VERDICTD_MAINTAINER)"
 
 clean:
-	@rm -rf $(Opa_Lib_Path)/libopa.*
+	cd $(Rats_Tls_Src_Dir) && make -C build uninstall
 	cargo clean && rm -f Cargo.lock
-	cd $(PROJDIR)/rats-tls && make -C build clean
-	@rm -f dist/rpm/verdictd.spec dist/deb/debian/changelog
\ No newline at end of file
+	@rm -rf $(Opa_Lib_Path)/libopa.*
+	@rm -f dist/rpm/verdictd.spec dist/deb/debian/changelog
+	@rm -fr $(Rats_Tls_Src_Dir)
```

Comment:
I think you can preserve the source code of rats-tls. Instead, use distclean rule to delete rats-tls:

.PHONY: all clean distclean build_rats_tls

distclean:
   $(MAKE) clean
   @rm -fr $(Rats_Tls_Src_Dir)